packages/compartment-mapper/src/types.js

```diff
@@ -239,6 +245,7 @@ export {};
  * @property {Array<Transform>} [transforms]
  * @property {Array<Transform>} [__shimTransforms__]
  * @property {Record<string, object>} [modules]
+ * @property {ExitModuleImportHook} [exitModuleImportHook]
```

Comment:
I'm wondering if 
```
* @property {Record<string, object> | ExitModuleImportHook} [modules]
```
would be frowned upon.

Reply:
I'd prefer not to mix types in this way in the API (policy description type mixing as you've done in the past is a different matter, for reasons of needing to be concise).  I'd rather have just an assertion around `!modules || !exitModuleImportHook`.

Reply:
I'm willing to remove the `modules` option entirely if you mark it as a breaking change, and update any uses within endo that depend on it to use the `exitModuleImportHook` instead.

Reply:
I see you're assuming this is more ready than it really is. 
I experimented with removing the modules option entirely and it does improve a bunch of things, but also has some interesting consequences. I'll talk about it in the meeting.